### ISSUE TITLE
missing PATO prefix

### ISSUE THREAD

**Issue by balhoff** [biolink/biolink-model#450]
Biolink model references PATO several times, but there is no PATO prefix in the context. I thought Biolink was merging in the OBO prefix set; is this not true?

**Comment by balhoff** [comment]
Same thing with `PCO`.

**Comment by deepakunni3** [comment]
@balhoff Yes, OBO context is being imported but somehow PATO and PCO is not being included in the generated JSON-LD context.

**Comment by balhoff** [comment]
`ENVO` is also missing from the JSON-LD context.

**Comment by deepakunni3** [comment]
I believe there is a bug in linkML where prefixes for CURIEs in certain slots are not imported from the upstream JSON-LD context. i.e. even if `PATO` and `PCO` exist in OBO context, somehow linkML doesn't see that it should be importing them.
